CWE/Modules/Community.cs

```diff
@@ -0,0 +1,137 @@
+                await Context.Channel.SendMessageAsync(embed: error);
+                return;
+            }
+
+            //List of 8-ball response strings
+            string[] responses =
+            {
+                //negitive responses
+                "My reply is no",
+                "Don’t count on it",
+                "Very doubtful",
+                "Outlook not so good",
+                "My sources say no",
+
+                //affirmative responses
+                "Yes",
+                "Most likely",
+                "It is certain",
+                "Signs point to yes",
+                "Outlook good",
+                "As I see it, yes",
+                "Yes, definitely",
+                "It is decidedly so",
+                "Without a doubt",
+                "You may rely on it",
+
+                //non commital responses
+                "Better not tell you now",
+                "Reply hazy try again",
+                "Concentrate and ask again",
+                "Cannot predict now",
+                "Ask again later"
+            };
+
+            var randomizedResponse = responses[new Random().Next(0, responses.Count())];
+            var eightBall = Embeds.eightBallEmbed(question, randomizedResponse, Context.User);
+            await Context.Channel.SendMessageAsync(embed: eightBall);
+        }
+
+        /// <summary>
+        ///     The command used to make a decision via coinflip.
+        /// </summary>
+        /// <returns>A <see cref="Task" /> representing the result of the asynchronous operation.</returns>
+        [Command("coinflip", RunMode = RunMode.Async)]
+        public async Task Coinflip()
+        {
+            //List of response strings
+            string[] coin =
+            {
+                "Heads",
+                "Tails"
+            };
+
+            var randomizedResponse = coin[new Random().Next(0, coin.Count())];
+            var coinflip = Embeds.CoinflipEmbed(randomizedResponse,
+                Context.User.Username + "#" + Context.User.Discriminator);
+            await Context.Channel.SendMessageAsync(embed: coinflip);
+        }
+
+        /// <summary>
+        ///     The command used to create a new community poll.
+        /// </summary>
+        /// ///
+        /// <param name="pollmsg">The <see cref="string" /> in the campaign.</param>
+        /// <returns>A <see cref="Task" /> representing the result of the asynchronous operation.</returns>
+        [Command("poll", RunMode = RunMode.Async)]
+        public async Task Poll([Remainder] string pollmsg)
+        {
+            var emoteUp = new Emoji("⬆️");
+            var emoteDown = new Emoji("⬇️");
+
+            var poll = Embeds.pollEmbed(pollmsg, Context.User);
+            var pollemb = await Context.Channel.SendMessageAsync(embed: poll);
+            await pollemb.AddReactionAsync(emoteUp);
+            await pollemb.AddReactionAsync(emoteDown);
+        }
+    }
+}
```

Comment:
Not in accordance with StyleCop.Analyzers (i.e. position of using directives). Weird spacing before the summary, why do commands run using `RunMode.Async` when they are quite simple?